commands/releases/output.js

```diff
@@ -0,0 +1,53 @@
+'use strict'
+
+let cli = require('heroku-cli-util')
+let co = require('co')
+let request = require('request')
+
+function * run (context, heroku) {
+  function getRelease (id) {
+    return heroku.get(`/apps/${context.app}/releases/${id}`)
+  }
+
+  function getLatestRelease () {
+    return heroku.request({
+      path: `/apps/${context.app}/releases`,
+      partial: true,
+      headers: { 'Range': 'version ..; max=2, order=desc' }
+    }).then((releases) => releases[0])
+  }
+
+  let release
+  if (context.args.release) {
```

Comment:
@dmathieu could you refactor this into a method in `lib` so that it can be shared between the two methods since they have the same arguments => release resolution?  https://github.com/heroku/heroku-apps/blob/master/commands/releases/info.js#L10-L21